src/haddock/libs/libontology.py

```diff
@@ -165,6 +165,12 @@ def check_faulty(self):
                 if element.is_present():
                     present += 1
 
+        if total == 0:
+            _msg = ("No outputs were expected to exist, "
```

Comment:
Not sure what this message intends to say.... No outputs were expected to exists - it sounds as if you don't expect any output

Reply:
that is right, if this gets triggered means you did not pass anything

Reply:
Then the message is misleading. Should rather read something like:
`None of the expected output was passed`

Reply:
Changed!